cmd/http-server/startcmd/start.go

```diff
@@ -95,6 +95,12 @@ const (
 		" Alternatively, this can be set with the following environment variable: " +
 		agentHTTPResolverEnvKey
 
+	// TODO Derive the SDS URL from the hub-auth bootstrap data #271
+	sdsURLFlagName      = "sds-url"
+	sdsURLFlagShorthand = "s"
+	sdsURLFlagUsage     = "URL SDS instance is running on."
+	sdsURLEnvKey        = "HTTP_SERVER_SDS_URL"
+
```

Comment:
This is just a temporary measure so that I can test SDS storage. Once hub-auth is ready, I'll update edge-agent to use that to get the SDS URL from the bootstrap data.